### PR TITLE
Add release notes for 0.16.5 and 0.16.6

### PR DESCRIPTION
**Pull request by benjaoming**
## Summary

Forgot this in 0.16.5, let's not do that for 0.16.6

### CODE REVIEW

**Comment by benjaoming** [comment]
Stuff has been failing consistently for some hours now, but this PR was okay before I did a force-push with a non-essential change.

File: data/version.yml [hunk 1]
```diff
@@ -1,3 +1,36 @@
+0.16.6:
+  release_date: "2016/06/14"
+  git_commit: ""
```
Comment (aronasorman) [inline data/version.yml]: Don't forget to update this once we're sure of a release!

**Comment by aronasorman** [comment]
Just one comment, merge when ready.

File: data/version.yml [hunk 2]
```diff
@@ -1,3 +1,36 @@
...
+  new_features: []
```
Comment (aronasorman) [inline data/version.yml]: Do you want to mention an updated content pack?

File: data/version.yml (hunk 1 above)
Comment (benjaoming) [inline data/version.yml]: I actually had a commit hash in the previous commit, but it was not the final one so removed it and force-pushed :)

I'll make amends during the release process.

**Comment by benjaoming** [comment]
Adding all this over in #5158

**Pull request closed without merging**